<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INWS03011-02" ignored="0" name="Test1" tests="2" failures="2" timestamp="2021-06-18T10:49:12 IST" time="8.626" errors="0">
  <testcase name="Test1" time="0.037" classname="com.ohrm.qa.testcases.OrangeHRMTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.ohrm.qa.pages.BasePage.sendKeys(BasePage.java:22)
at com.ohrm.qa.pages.LoginPage.enterUserName(LoginPage.java:17)
at com.ohrm.qa.testcases.OrangeHRMTest.Test1(OrangeHRMTest.java:27)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.005" classname="com.ohrm.qa.testcases.OrangeHRMTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.ohrm.qa.pages.BasePage.sendKeys(BasePage.java:22)
at com.ohrm.qa.pages.LoginPage.enterUserName(LoginPage.java:17)
at com.ohrm.qa.testcases.OrangeHRMTest.Test2(OrangeHRMTest.java:37)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test1 -->
